src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    private String name;
```

Comment:
remove this field, if you wanna add the name of the figure to the information in the draw() method just write it directly there